### PR TITLE
Hotkey improvements

### PR DESCRIPTION
**Pull request by SebastiaanKloos**
- Added better documentation inside the input.hotkeys.js file
- Added the Javascript API documentation to the UI documentation

### CODE REVIEW

File: modules/system/assets/ui/docs/input-hotkey.md [hunk 1]
```diff
@@ -18,4 +18,14 @@ Allows keyboard shortcuts (hotkeys) to be bound to an element's click event.
         Press "Shift + R" on your keyboard
     </button>
 
+## Javascript API
 
+```html
+
+$('html').hotKey({
+  hotkey: 'ctrl+s, cmd+s',
+  hotkeyVisible: false,
```
Comment (LukeTowers) [inline modules/system/assets/ui/docs/input-hotkey.md]: Please use four spaces for indentation and explain why the `hotkeyVisible` parameter is required.

Comment (SebastiaanKloos) [inline modules/system/assets/ui/docs/input-hotkey.md]: Committed some changes regarding your response.

File: modules/system/assets/ui/docs/input-hotkey.md [hunk 2]
```diff
@@ -18,4 +18,16 @@ Allows keyboard shortcuts (hotkeys) to be bound to an element's click event.
         Press "Shift + R" on your keyboard
     </button>
 
+## Javascript API
 
+If you use a selector other than a button or a link, you will need to add the `hotkeyVisible` property to the hotkey config.
+
+```html
```
Comment (LukeTowers) [inline modules/system/assets/ui/docs/input-hotkey.md]: The documentation doesn't utilize language hinting or backticks for code syntax highlighting, indent code examples by four spaces and remove the backticks and then we can merge this. Thanks for your work so far!

**Comment by daftspunk** [comment]
I don't understand why this makes a difference

```
$(document).ready(function() {
    $('html').hotKey({ hotkey: 'ctrl+s, cmd+s', callback: doSomething });
});
```

When I do this I get `true` so it shouldn't matter, should it?
```
$('html').is(':visible')
```

**Comment by SebastiaanKloos** [comment]
@LukeTowers I've changed my documentation regarding your response

**Comment by SebastiaanKloos** [comment]
@daftspunk I'ts a long time ago since I used the hotkey code. But everything I tried wasn't working until I added the 'hotkeyVisible'

**Pull request merged**